packages/swapkit/helpers/src/helpers/memo.ts

```diff
@@ -62,22 +72,41 @@ export const getMemoFor = <T extends MemoType>(memoType: T, options: MemoOptions
             return `${chain}.${symbol}`;
         }
       };
-
-      return singleSide
+      const memoWithoutAffiliate = singleSide
         ? `${memoType}:${chain}/${symbol}`
         : `${memoType}:${getPoolIdentifier(chain, symbol)}:${address || ""}`;
+
+      if (affiliateAddress && affiliateBasisPoints) {
```

Comment:
I would just test for affiliateAdress
what if someone just wants to track volume?